### PR TITLE
Merge 12.2 code freeze into trunk

### PR DESCRIPTION
**Pull request by spencertransier**
This PR merges the 12.2 code freeze changes into `trunk`. Includes: 
- Version bump
- Frozen app strings for translation
- Updated release notes
- Updated pods to the stable releases
- Updated the Sentry Fastlane plugin to the latest version

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -27,15 +27,15 @@ def aztec
...
+  pod 'Automattic-Tracks-iOS', '~> 2.0.0'
   # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :branch => 'trunk'
   # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :commit => ''
   # pod 'Automattic-Tracks-iOS', :path => '../Automattic-Tracks-iOS'
 end
 
 def wordpress_kit
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressKit', '~> 6.0-beta'
+  pod 'WordPressKit', '~> 6.0'
```
Comment (mokagio) [inline Podfile:38]: I think our existing logic in release-toolkit warns if there's a `-beta` prefix in the version rule, but it should be noted that leaving it there doesn't hurt.

Running `bundle exec pod update WordPressKit` with the rule `~> 6.0-beta`, for example, would have resulted in version 6.0.0 being installed.

**Review by mokagio — APPROVED** [review]
- [x] Update internal Pods to stable version
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `Deliverfile`
- [x] `AppStoreStrings.pot` updated accordingly to new version and release notes

File: Podfile (hunk 1 above)
Comment (mokagio) [reply Podfile:38]: E.g., this is what was run during the WordPress 21.7 code freeze, with the pods having the `-beta` suffix in the `Podfile`:

image

Comment (mokagio) [reply Podfile:38]: I'm not necessarily arguing for keeping the `-beta` prefix. Just wanted to point out this noteworthy CocoaPods behavior.

Comment (spencertransier) [reply Podfile:38]: Gotcha @mokagio, thanks for explaining! Do you usually only update the `Podfile` when it's necessary based on the version rule?

**Pull request merged**